src/callbacks.py

```diff
@@ -160,9 +161,15 @@ def _transform(self):
                 break
         self.model.train()
         average_losses = sum(partial_batch_losses) / steps
-        outputs = {'{}_prediction'.format(name): np.vstack(outputs_) for name, outputs_ in outputs.items()}
+        try:
```

Comment:
@apyskir I don' love using try catch here but handling it with lists here is probably not necessary.

Reply:
@jakubczakon I think that we can drop this try..except here, because batch_size=1 is required, so we can leave it as list.

Reply:
I was mistaken. `batch_size=1` is required only if crop_and_pad. Otherwise we can have `batch_size>1`.